src/ocm/services/apis/ClustersAPI.ts

```diff
@@ -96,7 +98,9 @@ const ClustersAPI = {
   },
 
   get(clusterId: Cluster['id']) {
-    return client.get<Cluster>(`${ClustersAPI.makeBaseURI(clusterId)}`);
+    return client.get<Cluster>(`${ClustersAPI.makeBaseURI(clusterId)}`, {
+      signal: controller.signal,
```

Comment:
IIUC this is not ensuring that the polling request will be always cancelled. This associates the signal and controller and whenever another get request is issued, the association to previous fetch is lost and replaced with this new one. Canceling the request will always cancel the last get request.

IIUC what we want instead is to create a dedicated AbortController when we are creating a poll get request and pass it's signal to the ClustersService which passes it to ClusterAPI which adds it to the `get` request as an option.

Aborting the polling request can be then passed as an action or function through context.

Reply:
I'll make a new PR to fix this behavior. I'll revert the AbortController changes